### PR TITLE
All awesome book

### PR DESCRIPTION
**Pull request by ademibowale**
- [ ] This is a pair program project 
- [ ] We refactor the project to add three new sections on top of the two sections we have already completed
- [ ] Added a new javascript  features t .
- [ ] Our  code follows the style guidelines of this project
- [ ] We have performed a self-review of my code
- [ ] We have commented my code, particularly in hard-to-understand areas
- [ ] We have made corresponding changes to the documentation
- [ ] Our changes generate no new warnings
- [ ] We have added tests that prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my changes

### CODE REVIEW

File: modules/saveBooksData.js [hunk 1]
```diff
@@ -0,0 +1,72 @@
...
+        const { id } = e.path[1];
+        for (let a = 0; a < this.booksStore.length; a += 1) {
+          if (this.booksStore[a].title === id) {
+            this.booksStore.splice(a, 1);
+            localStorage.setItem('data', JSON.stringify(this.booksStore));
+          }
+        }
+      }
+    });
+  }
```
Comment (Deepakdanger) [inline modules/saveBooksData.js:71]: - [ ] When trying to remove **type error** is coming due to which it's not removing from local storage. When I refresh the page all data were still there after removing as well. Please fix this.

Screenshot from 2022-07-01 00-19-38

**Review by Deepakdanger — CHANGES_REQUESTED** [review]
Hi @ademibowale & @mbonabucya, 

Good job so far! 👏
There are some issues that you still need to work on to go to the next project but you are almost there!

## To Highlight :100: 
- No linter error :heavy_check_mark: 
- Website matches the wireframe design :+1: 

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
- N/A

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: modules/saveBooksData.js (hunk 1 above)
Comment (ademibowale) [reply modules/saveBooksData.js]: @Deepakdanger please use chrome it working fine on chrome dont use firefox to view this project pls i have show it to 2 developers and has no issues pls .
thank you bro

**Review by dasileker — APPROVED** [review]
### **Hi @ademibowale,** 

### **Congratulations! 🎉**

Your project is complete!  There is nothing else to say other than... it's time to merge it 🚀 : 
 



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**